connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/PostgresIntegrationSuite.scala

```diff
@@ -389,24 +390,25 @@ class PostgresIntegrationSuite extends DockerJDBCIntegrationSuite {
     val df = sqlContext.read.jdbc(jdbcUrl, "char_types", new Properties)
     val row = df.collect()
     assert(row.length == 1)
-    assert(row(0).length === 5)
```

Comment:
May I ask why we remove this?

Reply:
we will check the columns one by one later，it seems an unnecessary assertion step

Reply:
For me, it's necessary because the JDBC source may return extra columns, @yaooqinn . Could you recover them?

Reply:
OK

Reply:
I refactored it to use `checkAnswer`